### PR TITLE
LPD-36442

### PR DESCRIPTION
**Pull request by NemethNorbert**
Hey,

Ticket: [LPD-36442](https://liferay.atlassian.net/browse/LPD-36442)

Root Cause:

Issue is caused by the fact we display the UndoRedo component, that was not designed to be displayed there. To fix it I hide it, when defaultValues are beeing edited.

How to test?
Follow ticket description [LPD-36442](https://liferay.atlassian.net/browse/LPD-36442)

Test included

### CODE REVIEW

**Review by DiegoHu97 — APPROVED** [review]
LGTM! Thank you :smile:

**Comment by DiegoHu97** [comment]
ci:forward

**Comment by DiegoHu97** [comment]
ci:stop

**Comment by DiegoHu97** [comment]
ci:test:content-management-playwright

**Comment by DiegoHu97** [comment]
ci: forward

**Comment by DiegoHu97** [comment]
ci:forward

**Comment by DiegoHu97** [comment]
ci:stop

File: modules/test/playwright/tests/journal-web/journalArticleAutosave.spec.ts [hunk 1]
```diff
@@ -35,7 +36,37 @@ const autoSaveTest = mergeTests(
 	}),
 	isolatedSiteTest,
 	journalPagesTest,
-	loginTest()
+	loginTest(),
+	pagesAdminPagesTest
+);
+
+autoSaveTest(
+	'LPD-36442 UndoRedo Should not appear when editing default values',
```
Comment (DiegoHu97) [inline modules/test/playwright/tests/journal-web/journalArticleAutosave.spec.ts:44]: Hey @NemethNorbert I just realised about this, could you move the LPD to the `@tag` naming?

**Review by DiegoHu97 — CHANGES_REQUESTED** [review]

**Comment by NemethNorbert** [comment]
ci:forward

**Review by DiegoHu97 — APPROVED** [review]

**Pull request closed without merging**